### PR TITLE
Updates to match template

### PR DESCRIPTION
**Pull request by heiskr**
Updates to match template

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,27 +16,28 @@
...
 - **What you'll build**: We will create two deployment workflows - the first workflow to deploy to staging based on a label and the second workflow to deploy to production based on merging to main.
-- **Prerequisites**: Before you start, you should be familiar with GitHub, GitHub Actions, and Continuous Integration with GitHub Actions. 
+- **Prerequisites**: Before you start, you should be familiar with GitHub, GitHub Actions, and Continuous Integration with GitHub Actions.
 - **How long**: This course is 6 steps long and takes less than 2 hours to complete.
 
 ## How to start this course
 
-1. Above these instructions, right-click **Use this template** and open the link in a new tab.
-   ![Use this template](https://user-images.githubusercontent.com/1221423/169618716-fb17528d-f332-4fc5-a11a-eaa23562665e.png)
+1. Right-click **Start course** and open the link in a new tab.
```
[the remark this answers is not in the document]
Comment (heiskr) [reply README.md:32]: It'll have both technically, but we've seen some learners struggle to find the "Use this template" button. The destination is the same though.

[the remark this answers is not in the document]
Comment (chadfawcett) [reply README.md:32]: Oh this is a button itself. Thought it was saying the "Use this template" button was somehow changed to "Start course". This makes sense, although I tested it on the "Connect the dots" repo and get a 404. Is there something missing to make this all work as expected?

[the remark this answers is not in the document]
Comment (chadfawcett) [reply README.md:32]: Looks like the dotcom interface making the link relative, even though you've started with `/`. i.e. for the Connecting the dots course it makes the link `https://github.com/skills/connect-the-dots/blob/main/skills/connect-the-dots/generate`

[the remark this answers is not in the document]
Comment (heiskr) [reply README.md:32]: Updated the link across the pull requests to absolute, ie start with `https://github.com`

**Review by chadfawcett — APPROVED** [review]

**Pull request merged**